app/_locales/hn/messages.json

```diff
@@ -0,0 +1,819 @@
+    "message": "किसी सार्वजनिक स्थान पर अपने बीज के शब्द ठीक नहीं करें! ये शब्द आपके सभी खातों को चोरी करने के लिए उपयोग किए जा सकते हैं।"
+  },
+  "revert": {
+    "message": "वापस"
+  },
+  "rinkeby": {
+    "message": "रिचीव टेस्ट नेटवर्क"
+  },
+  "ropsten": {
+    "message": "रॉप्स्टेन टेस्ट नेटवर्क"
+  },
+  "sampleAccountName": {
+    "message": "उदाहरण के लिए मेरा नया खाता",
+    "description": "उपयोगकर्ता को अपने खाते में मानव-पठनीय नाम जोड़ने की अवधारणा को समझें."
+  },
+  "save": {
+    "message": "सहेजें"
+  },
+  "saveAsFile": {
+    "message": "फ़ाइल के रूप में सहेजें",
+    "description": "खाता निर्यात प्रक्रिया"
+  },
+  "saveSeedAsFile": {
+    "message": "सेड वर्ड्स - फाईल्स सेव करें"
+  },
+  "search": {
+    "message": "खोज करें"
+  },
+  "secretPhrase": {
+    "message": "अपनी गुप्त बारह शब्द वाक्यांश यहाँ अपनी तिजोरी को पुनर्स्थापित करने के लिए दर्ज करें।"
+  },
+  "seedPhraseReq": {
+    "message": "बीज वाक्यांश 12 शब्द लंबा हैं"
+  },
+  "select": {
+    "message": "चुनें"
+  },
+  "selectCurrency": {
+    "message": "मुद्रा चुनें"
+  },
+  "selectService": {
+    "message": "सेवा चुनें"
+  },
+  "selectType": {
+    "message": "प्रकार चुनें"
+  },
+  "send": {
+    "message": "भेजें"
+  },
+  "sendETH": {
+    "message": "भेजें ETH"
+  },
+  "sendTokens": {
+    "message": "भेजें टोकन"
+  },
+  "sendTokensAnywhere": {
+    "message": "इटोरम खाते वाले किसी को भी टोकन भेजें"
+  },
+  "settings": {
+    "message": "सेटिंग्स"
+  },
+  "shapeshiftBuy": {
+    "message": "शेपेशिस्ट के साथ खरीदें"
+  },
+  "showPrivateKeys": {
+    "message": "निजी कुंजी दिखाएँ"
+  },
+  "showQRCode": {
+    "message": "QR कोड दिखाएं"
+  },
+  "sign": {
+    "message": "हस्ताक्षर"
+  },
+  "signMessage": {
+    "message": "हस्ताक्षर संदेश"
+  },
+  "signNotice": {
+    "message": "इस संदेश पर हस्ताक्षर करने से \n साइड इफेक्ट हो सकते हैं। \n केवल अपने पूरे खाते के साथ पूरी तरह से भरोसेमंद \n साइटों से संदेश पर हस्ताक्षर करें। \n यह खतरनाक विधि भविष्य के संस्करण में निकाल दी जाएगी।"
+  },
+  "sigRequest": {
+    "message": "हस्ताक्षर अनुरोध"
+  },
+  "sigRequested": {
+    "message": "हस्ताक्षर अनुरोधित"
+  },
+  "spaceBetween": {
+    "message": "केवल शब्दों के बीच एक स्थान हो सकता है"
+  },
+  "status": {
+    "message": "स्थिति"
+  },
+  "stateLogs": {
+    "message": "स्थिति संदेश"
+  },
+  "stateLogsDescription": {
+    "message": "स्थिति संदेश में आपका सार्वजनिक खाता, पतों और भेजे गए लेनदेन, होते हैं।"
+  },
+  "submit": {
+    "message": "सबमिट करें"
+  },
+  "supportCenter": {
+    "message": "हमारे सहायता केंद्र पर जाएं"
+  },
+  "symbolBetweenZeroTen": {
+    "message": "प्रतीक 0 और 10 अक्षरों के बीच होना चाहिए"
+  },
+  "takesTooLong": {
+    "message": "बहुत समय ले रहा है?"
+  },
+  "terms": {
+    "message": "उपयोग की शर्तें"
+  },
+  "testFaucet": {
+    "message": "टेस्ट नलि"
+  },
+  "to": {
+    "message": "के लिए"
+  },
+  "toETHviaShapeShift": {
+    "message": "शैपशैफ्ट द्वारा $ 1 से ETH में जमा",
```

Comment:
`$1` is a variable and must be written exactly this way

Reply:
@kumavis - [x] Made the `$1` behave as `variable`.
---
Any further changes required? Review me!